users/templates/user_notifications_list.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+
+{% load i18n %}
+{% load data_structure_element_selectors %}
+
+{% block content %}
+<h1>
+    &nbsp; {% trans 'Notifications' %} &nbsp;
```

Comment:
Why the extra spaces? :)

Reply:
I just copied it from other template :) Fixed :)

Reply:
Still there :)

Reply:
Now I don't really know about which spaces are you talking about? :P

Reply:
What is the `&nbsp;` there then? :)

Reply:
What about this one? `&nbsp;` mean non-breaking space in HTML. I am not saying its wrong to add, I was just curious why you have added them?

Reply:
To be honest I didn't know what it is and what it does. I just copied it from different template. I see now what it is for and this is not needed here. Deleting :)